include/CppUTestExt/MockNamedValue.h

```diff
@@ -82,6 +82,7 @@ class MockNamedValue
     virtual void setValue(void* value);
     virtual void setValue(const void* value);
     virtual void setValue(const char* value);
+    virtual void setValue(const unsigned char* value);
```

Comment:
I meant, it is probably better not to use the same sane (SetValue) for it, to avoid accidental errors.